components/director/internal/open_resource_discovery/service_test.go

```diff
@@ -48,6 +56,17 @@ func TestService_SyncORDDocuments(t *testing.T) {
 		return persistTx, transact
 	}
 
+	successfulLabelRepo := func() *automock.LabelRepository {
+		labelRepo := &automock.LabelRepository{}
+		labelRepo.On("ListGlobalByKeyAndObjects", txtest.CtxWithDBMatcher(), model.ApplicationLabelableObject, mock.Anything, applicationTypeLabel).Return([]*model.Label{
```

Comment:
You can make a custom mockery matcher that matches slices with just one element instead of mock.Anything. Because you return just one label. This will make the test a little bit more exhaustive.